### PR TITLE
Add jupyter language multi packs recipe

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- ~~When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.~~

This is based on https://github.com/conda-forge/staged-recipes/pull/15771#issuecomment-891248228

Closes #15771
Closes #15776
Closes #15777
Closes #15778
Closes #15779
Closes #15780
Closes #15781
Closes #15782
Closes #15783
Closes #15784
Closes #15785
Closes #15786
Closes #15787
Closes #15788
Closes #15789
Closes #15790
Closes #15791
Closes #15792
Closes #15793
Closes #15794
Closes #15795
Closes #15796
Closes #15797
Closes #15798
Closes #15799
Closes #15800
Closes #15801

### CODE REVIEW

**Comment by fcollonval** [comment]
Link to pipeline: https://dev.azure.com/conda-forge/feedstock-builds/_build?definitionId=2259&_a=summary

File: recipes/juypterlab-language-packs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - echo "no metapackage"
+
+outputs:
+{% for locale in locales %}
```
Comment (xhochy) [inline recipes/juypterlab-language-packs/meta.yaml:33]: This is going to mess a lot with automation. An alternative would be to list the locals in `conda_build_config.yaml`. That will work better with the bots but produce a really large CI matrix.

File: recipes/juypterlab-language-packs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+source:
+  - folder: ar-SA
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-ar-SA/{{ name[-1] }}-ar-SA-{{ version }}.tar.gz
+    sha256: d0e4a6d9ceeee573761e34731a6426a193956f7102ddfd40ae04620edb3dd61d
+  - folder: no-NO
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-no-NO/{{ name[-1] }}-no-NO-{{ version }}.tar.gz
+    sha256: 3f1bcaa8d8c40d5e1cb3051a78c584203ade87286cff492fc19878dc1027e9f5
+
+build:
+  number: {{ build_number }}
```
Comment (xhochy) [inline recipes/juypterlab-language-packs/meta.yaml:19]: Don't template the build number. if not specified in the output, the top level one is taken.
```suggestion
  number: 0
```

File: recipes/juypterlab-language-packs/meta.yaml (hunk 1 above)
Comment (fcollonval) [reply recipes/juypterlab-language-packs/meta.yaml:33]: Thanks for the early feedback @xhochy 
The for-loop is not working - locally it ends up with no outputs in the recipe. I'll give it a try with `conda_build_config`.

File: recipes/juypterlab-language-packs/meta.yaml (hunk 2 above)
Comment (fcollonval) [reply recipes/juypterlab-language-packs/meta.yaml:19]: Thanks for the tip

**Comment by fcollonval** [comment]
This change the conda packaging of JupyterLab language packs in a mono-recipe.

@blink1073 @goanpeca @krassowski do you agree to become maintainer of this recipes?

**Comment by fcollonval** [comment]
Pipeline on the latest commit are passing: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=366383&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd

**Comment by blink1073** [comment]
👍🏼

**Comment by krassowski** [comment]
I agree.

File: recipes/juypterlab-language-packs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-CN/{{ name }}-zh-CN-{{ version }}.tar.gz
+    sha256: cef77cc2fa9d4a56fbfcd47ee344c80fb807e3abdf94ef7d80312713a607a067
+  - folder: zh-TW
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-TW/{{ name }}-zh-TW-{{ version }}.tar.gz
+    sha256: 8b96f81306be0806ed92d3b6f9bec219c79e76b8c1b99deb56bb1fcf3028a6ec
+
+build:
+  number: 0
+  noarch: python
+  script: cd {{ locale }} && {{ PYTHON }} -m pip install . -vv --no-deps
```
Comment (xhochy) [inline recipes/juypterlab-language-packs/meta.yaml:94]: This should also work
```suggestion
  script: {{ PYTHON }} -m pip install {{ locale }} -vv --no-deps
```

File: recipes/juypterlab-language-packs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+about:
+  home: https://github.com/jupyterlab/language-packs
+  summary: JupyterLab {{ locale }} Language Pack
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: {{ locale }}/LICENSE.txt
+  doc_url: https://jupyterlab.readthedocs.io/
+  dev_url: https://github.com/jupyterlab/language-packs
+
+extra:
```
Comment (xhochy) [inline recipes/juypterlab-language-packs/meta.yaml:118]: Please explicitly add the desired feedstock name here.
```suggestion
extra:
  feedstock-name: jupyterlab-language-packs
```

File: recipes/juypterlab-language-packs/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-CN/{{ name }}-zh-CN-{{ version }}.tar.gz
+    sha256: cef77cc2fa9d4a56fbfcd47ee344c80fb807e3abdf94ef7d80312713a607a067
+  - folder: zh-TW
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-zh-TW/{{ name }}-zh-TW-{{ version }}.tar.gz
+    sha256: 8b96f81306be0806ed92d3b6f9bec219c79e76b8c1b99deb56bb1fcf3028a6ec
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install {{ locale }} -vv --no-deps
```
Comment (xhochy) [inline recipes/juypterlab-language-packs/meta.yaml:94]: Seems like somehow the suggestion to merge `cd` and `pip` did break the build :(

Comment (fcollonval) [reply recipes/juypterlab-language-packs/meta.yaml]: Oups of course pip is looking for a package named as the subfolder and for a local directory. I'll push a fix.

Comment (fcollonval) [reply recipes/juypterlab-language-packs/meta.yaml]: @xochy this is now fixed.

Comment (fcollonval) [reply recipes/juypterlab-language-packs/meta.yaml]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=366658&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

**Comment by blink1073** [comment]
Thanks @fcollonval and @xhochy!

**Pull request merged**